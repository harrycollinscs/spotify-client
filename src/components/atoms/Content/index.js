import React from "react";
import styled from "styled-components";

const ContentContainer = styled.div`
  padding: 30px 30px 30px 30px;
`;

const Content = ({ children }) => (
  <ContentContainer>
    {children}
  </ContentContainer>
);

export default Content;
