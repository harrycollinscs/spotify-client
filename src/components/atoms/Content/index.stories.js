import React from "react";
import Content from "./index";

export default {
  title: "Atoms/Content",
  component: Content,
};

const Template = () => <Content />;

export const Default = Template.bind({});
