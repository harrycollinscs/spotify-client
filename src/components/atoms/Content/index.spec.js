import React from "react";
import Content from "./index";
import { shallow } from "enzyme";

describe("Atoms/Content", () => {
  it("Should match snapshot", () => {
    const render = shallow(<Content />);
    expect(render).toMatchSnapshot();
  });
});
