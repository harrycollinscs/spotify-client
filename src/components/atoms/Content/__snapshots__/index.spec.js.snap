// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms/Content Should match snapshot 1`] = `<styled.div />`;
